theme/templates/resource-landing-page/top-right-buttons.html

```diff
@@ -11,7 +11,7 @@
             </a>
         {% endif %}
         {% if not resource_edit_mode %}
-            {% if not is_owner_user and not is_edit_user and not is_view_user and cm.raccess.public %}
+            {% if not self_access_level == 'None' and cm.raccess.public %}
                 {% if resource_is_mine %}
```

Comment:
Is "Shareable" addressed here and elsewhere? I never fully understood that part. 
![image](https://user-images.githubusercontent.com/45397838/58665395-c4688900-82fe-11e9-8964-e663ba967893.png)

Reply:
Shareable is addressed here and has been moved to the Vue instance:
https://github.com/hydroshare/hydroshare/blob/3186-manage-access-vue-instance/theme/templates/resource-landing-page/manage-access.html#L207